account_payment_confirm/models/account_payment.py

```diff
@@ -1,18 +1,10 @@
-# -*- coding: utf-8 -*-
-
-from openerp import api, models, _
-from openerp.exceptions import ValidationError
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
 
 
 class AccountPayment(models.Model):
     _inherit = "account.payment"
 
-    @api.multi
-    def check_post(self):
-        new_context = dict(
-            self.env.context, **{'account_payment_confirm': True})
-        self.with_context(new_context).post()
-
```

Comment:
Hmmm, why are you removing this method? Are we sure it's not used, even by other modules in the foodcoops repository?

I'm all in favour of removing useless code, but I'm worried this may have an undesireable impact.
Can you please confirm this method is not used?

Reply:
@ivantodorovich 
yes `check_post` was only use in  `account_payment_confirm`